Dockerfile.frontend-test

```diff
@@ -0,0 +1,17 @@
+# nb. We use Alpine as a base image and then install Node + Yarn separately
```

Comment:
I don't think we have a convention for naming auxiliary Dockerfiles in our repos. I'm happy to take suggestions. The reason I'm not just using `alpine:3.10` and installing the dependencies within the image in Jenkins on every run is because this approach makes the build a bit faster / lighter on Jenkins resources by benefiting from Docker layer caching.

Reply:
I did notice that this is `frontend-test` whereas the make command is `frontend-tests`